concurrency/DurableLocalStoreManagerImpl.java

```diff
@@ -0,0 +1,287 @@
+        final DurableLocalStore dataStore = bucketDataMap.get(bucketName);
+        dataStore.assertIfNotOpen("backup");
+        dataStore.backup(outputDir);
+    }
+
+    public boolean importLocalStore(String importDir, String bucketName) {
+        throw new UnsupportedOperationException(this.getClass().getName() + " does not support importLocalStore()");
+    }
+
+    @Override
+    public DurableLocalStore exportLocalStore(String bucketName, String outputDir) throws StorageNodeException {
+        backup(bucketName, outputDir);
+        return new DurableLocalJsonStore(outputDir, bucketName, false);
+    }
+
+    @Override
+    public DurableLocalStore getDurableLocalStore(String bucketName) {
+        return bucketDataMap.get(bucketName);
+    }
+
+    @Override
+    public void cleanupForUnitTestOnly(boolean doCleanupData) {
+        //to allow for deletes from bucketDataMap since safeClose modifies it.
+        Set<String> buckets = new HashSet<>(this.bucketDataMap.keySet());
+        for(String bucketName : buckets) {
+            safeClose(bucketName);
+            if(doCleanupData) {
+                dropSingleStoreForUnitTestOnly(bucketName);
+            }
+        }
+        this.bucketDataMap.clear();
+    }
+    
+    @Override
+    public void dropAllStoresForUnitTestOnly() {
+        for (final StorageInstance si :  sim.getDataStorageInstances()) {
+            File dataDir = new File(si.instanceConfiguration.getEngineConfig().getDataDir());
+            if(dataDir.exists() && dataDir.isDirectory()) {
+                for(File file : dataDir.listFiles()) {
+                    if(DurableLocalStore.isDurableStoreName(file.getName())) {
+                        dropSingleStoreForUnitTestOnly(sim.dbPrefix, dataDir.getPath(), file.getName());
+                    }
+                }
+            }
+        }
+    }
+
+    protected void dropSingleStoreForUnitTestOnly(String bucketName) {
+        final String dataFolder = sim.getStorageInstance(bucketName).instanceConfiguration
+        .getEngineConfig().getDataDir();
+        final String storeName = DurableLocalStore.getStoreName(bucketName);
+        
+        dropSingleStoreForUnitTestOnly(sim.dbPrefix, dataFolder, storeName);
+    }
+    
+    private void dropSingleStoreForUnitTestOnly(String dbPrefix, String dataFolder, String storeName) {
+
+        //delete only if the store / file belongs to this dbPrefix. Else we mess with parallel nodes.
+        if(! storeName.startsWith(dbPrefix)) {
+            return;
+        }
+        
+        // cleanup both kinds possible here.
+        try {
+            WiredTigerWrapper.getInstance(dbPrefix).dropDatabase(new Pair<>(dataFolder, storeName), true);
+        } catch (WiredTigerException e) {
+            // eat the stack trace if this fails - test cleanup
+        }
+
+        // for json files and WT folder - generic drop
+        new File(dataFolder, storeName).delete();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of deleting a file on this line should be checked and failure should be logged. java.io.File.delete() will simply return false if it fails to delete the file, for example, when the file does not exist. It will not throw unless there is a security exception.